docs/omero_figure_move.rst

```diff
@@ -0,0 +1,103 @@
+Move Figures between Groups
+===========================
+
+Moving Figures between Groups in OMERO might be necessary for example in cases where a scientist decides to move all their data into another Group to enable cooperation and viewing by other colleagues or to a Group which is to be made public later. 
```

Comment:
"to be made public later" is a bit too specific to our nightshade workflow. Why not a group that is already public?
Maybe just "...viewing by other colleagues or to make the data public".
Maybe remove the "all" earlier in the sentence.